lib/ansible/modules/files/tempfile.py

```diff
@@ -55,6 +55,13 @@
   tempfile:
     state: file
     suffix: temp
+  register: tempfile_1
+
+- name: remove the temporary file
+  file:
+    path: "{{ tempfile_1.path }}"
+    status: absent
```

Comment:
Thanks @tpo for adding an example. I think this line should be `state: file` (not `status`). Could you update the new example? (Or if I'm wrong and it really should be `status`, could you update the other examples so they match?) Thanks!

Reply:
Oh, hang on, I mis-read what your proposed example does. 

All examples in the `tempfile` module documentation should illustrate the use of the `tempfile` module - if you want to show that removing a temporary file can be accomplished using a different module (the `file` module), a note would be better than an example.

Reply:
Hi Alicia,

what I'm trying to illustrate is the use of the "return value", specifically of `return_value.path`. The table in the tempfile documentation describing the return value is not very evident (i.e. there's a column: "Returned": "success", the meaning of which is not clear without further explanation, an explanation that which is missing nota bene). That's the goal of my patch: showing in an example how to access the path of the tempfile in the return value. I think it's secondary *how* the value is used in the example (say file: status=absent) but the example *should* make some sense.

So additionaly to showing how the return value of tempfile is used, which is the main goal, the secondary goal - having an example that actually makes sense - is also fulfilled, by removing the temporary file after creation - something that most users will be able to copy paste as is, since as a rule you will want to remove the tempfile at some later point.

I see that you are requesting a change. Now I guess this patch will not be merged as long as there's this change request? In which case I'd request from you to please remove that request (?) if my explanation above makes sense.

Thanks,
*t